cli/cmd/install.go

```diff
@@ -132,9 +134,28 @@ A full list of configurable values can be found at https://www.github.com/linker
 					}
 					os.Exit(1)
 				}
+
+				// Initialize the k8s API which is used for the proxyInit
+				// runAsRoot check.
+				k8sAPI, err = k8s.NewAPI(kubeconfigPath, kubeContext, impersonate, impersonateGroup, 30*time.Second)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Create values override
+			valuesOverrides, err := options.MergeValues(nil)
+			if err != nil {
+				return err
+			}
+			runAsRoot := getRunAsRoot(valuesOverrides)
+			err = healthcheck.CheckProxyInitRunsAsRoot(cmd.Context(), k8sAPI, runAsRoot)
```

Comment:
should this be inside the `!ignoreCluster` block?  if ignore cluster is set, k8sAPI will be nil, right?